CombinationGenerator.py

```diff
@@ -0,0 +1,60 @@
+# =============================================================================
+#  Name:            CombinationGenerator
+#  Purpose:         
+#   
+#  Author:          Winnie Ding
+#  Date Created:    10/29/2020
+# =============================================================================
+
+
+def CombinationGenerator(a, b, c, d):
+    """
+    Takes 4 values and generates every combination to test in
+    reverse polish notation
+    """
+
+    values_list = [a, b, c, d]
+    operators_list = ['+', '-', '*', '/']
+
+
+    import itertools
+    #Generate permutations of 4 numbers
+    values_perms = []
+    for perm in itertools.permutations(values_list,4):
+        values_perms.append(perm)
+
+    #Generate combinations of 3 operators
+    operators_combs = []
+    for comb in itertools.combinations_with_replacement(operators_list,3):
+        operators_combs.append(comb)
+
+    #Generate orders of values vs operators
+    #Hard coded patterns
```

Comment:
Maybe some more documentation -- eg since we have exactly four operands, there are limited ways to arrange them into a valid expression...